apps/admin-page/src/app/nz-modules.constant.ts

```diff
@@ -1,4 +1,19 @@
 import { NzButtonModule } from 'ng-zorro-antd/button';
 import { NzGridModule } from 'ng-zorro-antd/grid';
+import { NzLayoutModule } from 'ng-zorro-antd/layout';
+import { NzBreadCrumbModule } from 'ng-zorro-antd/breadcrumb';
+import { NzMenuModule } from 'ng-zorro-antd/menu';
+import { NzIconModule } from 'ng-zorro-antd/icon';
+import { NzDividerModule } from 'ng-zorro-antd/divider';
+import { NzTypographyModule } from 'ng-zorro-antd/typography';
 
-export const nzModules = [NzButtonModule, NzGridModule];
+export const nzModules = [
+  NzButtonModule,
+  NzGridModule,
+  NzLayoutModule,
+  NzBreadCrumbModule,
```

Comment:
I believe that the Breadcrumb module is not needed and can be removed since it is not used anywhere in the layout
```suggestion
```